src/main/java/seedu/weeblingo/storage/LocalDatabasePopulator.java

```diff
@@ -38,6 +38,25 @@
         return flashcards;
     }
 
+    /**
+     * Gets the specified number of flashcards, as an array of flashcards.
+     *
+     * @return An array of flashcards of specified size extracted from database.
+     */
+    public static Flashcard[] getDatabaseOfFlashcards(int numberOfQuestions) {
```

Comment:
LGTM but having a separate method like:
```java
public static Flashcard[] getDatabaseOfFlashcards(int numberOfQuestions) {
    return Arrays.copyOfRange(getDatabaseOfFlashcards(), 0, numberofQuestions);
}
```
is nicer as my pending PR has many tests related to this method so I would imagine there would be lesser conflicts if you have the above instead. Thanks!!!